packages/driver/src/cypress/screenshot.coffee

```diff
@@ -16,6 +16,37 @@ defaults = reset()
 
 validCaptures = ["fullPage", "viewport", "runner"]
 
+normalizePadding = (padding) ->
+  padding ||= 0
+
+  if _.isArray(padding)
+    # CSS shorthand
+    # See: https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Tricky_edge_cases
+    switch padding.length
+      when 1
```

Comment:
As I said to you in DM, since we removed the extra `else` I still wonder whether it would be better to handle lengths that are zero or greater than 4. My understanding is that they will be rewriting in TypeScript [at some point](https://github.com/cypress-io/cypress/issues/2690), so it would be better if this function always returns `[number, number, number, number]` (even if the earlier validations would have made it impossible for other array lengths to be passed in).